### ISSUE TITLE
Non-member able to execute proposal 

### ISSUE THREAD

**Issue by Filipv95** [DXgovernance/DAVI#283]
**Description**
Executed Proposal as a non-member of SWPR Guild 

**Steps to Reproduce**
1. SpinUp Localhost env
2. Open localhost:3000
3. Check if Landing Page is loaded
4. Go to SWPR Guild page as a non-member of the Guild
5. Find Proposal with 'Executable' state
6. Execute the Proposal

**Expected behavior**
Only members of the Guild should be able to execute Proposals.

**Screenrecord**
https://user-images.githubusercontent.com/37701286/188395290-79aa8d47-085d-4abc-acf6-24c4dd164827.mp4

**Device Information**
OS: MacOs
Browser: GoogleChrome, Brave
Device: MacOS Monterey v12.4

**Comment by rossneilson** [comment]
I dont think we require users to be members on the smart contract side so this is the expected behaviour
